lib/resources/r4/procedure.yaml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
You are missing status

Reply:
Added [here](https://github.com/cerner/fhir.cerner.com/pull/322/commits/783dce5f317c0ee97b87117e79fb62dcbca23583#diff-80d4a0c17acadba0cd26a39c2cfeaf8dR5).

Reply:
+1